src/lib/accordion_item.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+  import { base } from '$app/paths';
+  import Button from '$lib/button.svelte';
+  import type { RowData } from '$lib/types';
+  import { onMount } from 'svelte';
+  import { slide } from 'svelte/transition';
+
+  let el: HTMLDivElement | null;
+  let h = 0;
+  export let expanded = true;
+  export let onClose: () => void;
+  export let rowData: RowData;
+  export let index: number;
+  let { imageName, imageAlt, extraContent, resourceTexts, resourceLinks } = rowData;
+
+  const COMMON_IMAGE_PATH = 'roadmap/steps';
+
+  onMount(() => {
+    if (el) {
+      h = el.offsetHeight;
+      onClose();
+    }
+  });
+</script>
+
+{#if expanded}
+  <div
+    bind:this={el}
+    style="max-height: {h === 0 ? 'auto' : expanded ? h : 0}px; display: {expanded
+      ? 'block'
+      : 'none'}"
+    transition:slide={{ duration: h }}
+    class="accordion-item"
+  >
+    <div>
+      <img src="{base}/{COMMON_IMAGE_PATH}/{imageName}" alt={imageAlt} class="accordion-image" />
+      <p class:first={index == 0}>{@html extraContent}</p>
+    </div>
+    {#each resourceTexts as resourceText, j}
+      <div class="resource-link">
+        <a href={resourceLinks[j]} class="link-no-underline"
+          ><Button fill={true}
+            ><div class="button-contents">
+              <img
+                src="{base}/icons/ic_external_link.svg"
+                alt="External Link"
+                class="external-link-icon"
+              />
+              {resourceText}
+            </div></Button
+          ></a
+        >
+      </div>
+    {/each}
+  </div>
+{/if}
+
+<style>
+  @media screen and (min-width: 675px) {
```

Comment:
when shrinking and growing, the modal accordion pops up before the mobile "get involved" and "cat stories" section. Not sure if this is because there are discrepancies between what is ok for tablet mode or not but just thought I'd mention